scalding-repl/src/main/scala/com/twitter/scalding/ScaldingILoop.scala

```diff
@@ -77,14 +82,16 @@ class ScaldingILoop
    */
   override def commands: List[LoopCommand] = super.commands ++ scaldingCommands
 
+  protected def shellImports: List[String] = List(
```

Comment:
could this be moved to BaseScaldingShell so there is only one place to configure if it makes sense?